apps/api/src/app/workflows-v2/usecases/build-payload-schema/build-payload-schema.usecase.ts

```diff
@@ -64,7 +67,7 @@ export class BuildPayloadSchema {
 
     for (const [key, value] of Object.entries(controlValue)) {
       if (isStringTipTapNode(value)) {
-        processedValue[key] = transformMailyContentToLiquid(JSON.parse(value));
+        processedValue[key] = this.hydrateEmailSchemaUseCase.execute({ emailEditor: value });
```

Comment:
I realized `HydrateEmailSchemaUseCase` does the same as `transformMailyContentToLiquid` 

I think at some point we should merge the `HydrateEmailSchemaUseCase` and `ExpandEmailEditorSchema` to single usecase - there is no reason to transform Maily JSON to Liquid in multiple usecases like `Maily` -> `Hydrate` -> `Expand` -> `ParseByLiquid`.

It would be better to have one usecase which gets Maily JSON and outputs HTML parsed by Liquid.

Reply:
I agree 100%